src/components/AddAnimal.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React from "react";
```

Comment:
Is there a difference between " and  ' ?

Reply:
I think "format document" in VSCode did it 😄